### DISCUSSION TITLE
Seems the plugin document is outdated

### DISCUSSION THREAD

**Discussion by vip30** [swc-project/swc#5619] [General]
I'm trying to follow the tutorial here but seems swc_plugin already been deprecated https://swc.rs/docs/plugin/ecmascript/getting-started

is that possible to update the document for those not familar with swc? 

Thanks

**Comment by kdy1** [comment]
Seems like https://github.com/swc-project/website/commit/d83093c9090b902051ec06b060e98f8d433d768b is not published?

**Reply by kdy1** [reply]
Seems like it's caused by renaming branch from master to main

**Reply by kdy1** [reply]
Now it's fixed
